app/Http/Controllers/RestaurantManagementController.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use \Illuminate\Validation\ValidationException;
+
+use App\Services\RestaurantManagementService;
+
+class RestaurantManagementController extends Controller
+{
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+    }
+
+    /**
+     * 店舗の一覧取得 (owner, counter, kitchenは自分の店舗のみ取得可能).
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function restaurantList()
+    {
+        $restaurants = RestaurantManagementService::getRestaurantList();
+        // 店舗毎のowner_admin_idを取得し、レスポンスに追加
+        foreach ($restaurants as $restaurant) {
+            $owner = RestaurantManagementService::getOwner($restaurant->restaurant_id);
+            if ($owner !== null) {
+                $restaurant['owner_admin_id'] = $owner->admin_id;
+            } else {
+                $restaurant['owner_admin_id'] = null;
+            }
```

Comment:
```suggestion
            $restaurant['owner_admin_id'] = $owner?->admin_id;
```
[MAY] **php**
nullセーフ演算子というのを使うとスッキリします！
https://www.php.net/manual/ja/language.oop5.basic.php#language.oop5.basic.nullsafe